dataprofiler/profilers/data_labeler_column_profile.py

```diff
@@ -238,6 +239,34 @@ def profile(self):
         }
         return profile
 
+    def diff(self, other_profile, options=None):
+        """
+        Generates the differences between the orders of two DataLabeler columns
+
+        :return: Dict containing the differences between orders in their
+        appropriate output formats
+        :rtype: dict
+        """
+        cls = self.__class__
+        if not isinstance(other_profile, cls):
+            raise TypeError("Unsupported operand type(s) for diff: '{}' "
+                            "and '{}'".format(cls.__name__,
+                                              other_profile.__class__.__name__))
```

Comment:
Can you call the base class for the error check? You can use the returned empty dict for the differences dict